src/App.vue

```diff
@@ -743,8 +743,8 @@ export default class App extends Mixins(
       }
 
       // ensure types match
-      if (this.nrTypeToEntityType(nrResponse) !== this.getEntityType) {
-        console.log('NR type doesn\'t match entity type') // eslint-disable-line no-console
+      if (nrResponse.legalType !== this.getEntityType) {
+        console.log('NR legal type doesn\'t match entity type') // eslint-disable-line no-console
```

Comment:
nice! 
so essentially no more local mapping required?

Reply:
Correct. It's now done in the API before the UI gets the data. (It's a new field in the NR JSON.)

Reply:
Very nice!